### ISSUE TITLE
nodejs package - Authorisation Through demoapi not working because query-string not form

### ISSUE THREAD

**Issue by Gwillink** [CurrencyCloud/currencycloud-js#3]
in the currency-cloud package for nodejs, in the lib => "client.js" file. There is an issue on calling "requestToken" function since 22.11.2017 about 3pm CET. The issue is as marked in the following:
Note* : only in DEMO environment.

Seems like there is an issue with "qs", i tried with "form" and it worked

```
var requestToken = function() {
  var promise = rp.post({
    uri: config.baseUrl + config.authUrl,
   qs: {
      login_id: config.loginId,
      api_key: config.apiKey
    }
  })
  .then(function(res) {
    token = JSON.parse(res).auth_token;

    return token;
  });
  return promise;
};
```

**Comment by ermi-ltd** [comment]
This issue is now present on live. The suggested fix above (changing from qs to form) is working for us too.

**Comment by EAddario** [comment]
As per comment above, this was addressed in v0.10.0
